README.md

```diff
@@ -0,0 +1,13 @@
+BeCheckers
+=
+
+![BeCheckers Screenshot](BeCheckers.png)
+
+BeCheckers is a simple checkers game developed for two players.  The game conforms to almost all ACF (American Checker Federation) rules, including:
+
+	*	No huffs (one must complete all jumps in a jump sequence)
```

Comment:
Remove the tab in front of the "*" and just a simple space after it. Otherwise the formatting is off.